INSTALL.md

```diff
@@ -0,0 +1,121 @@
+![image](https://user-images.githubusercontent.com/2686765/156219166-0ad2a46f-a3c4-4d4e-b1ab-d72ad0d7b8c4.png)
+![image](https://user-images.githubusercontent.com/2686765/156219190-1bda80c0-fe77-42ae-9d9c-d10bebd422e1.png)
+![image](https://user-images.githubusercontent.com/2686765/156219210-ba23cf55-f0c2-4734-8a1c-6acb865fbc49.png)
+
+Now we're done!
+![image](https://user-images.githubusercontent.com/2686765/156219231-c051bb0f-a816-4dfe-871a-7e4edb6c4780.png)
+![image](https://user-images.githubusercontent.com/2686765/156219254-c00b5fe9-53c7-411d-88c2-c6cf798fa665.png)
+
+Note: there may be a screen asking if we want to install with a UEFI-based bootloader. Say yes. Afterwards, continue through the rest of the install as normal. 
+
+<h2>First boot</h2>
+Since Debian does not have the same packages as Ubuntu (i.e. no PPAs), some modifications are needed from the usual VxSuite build process. First, you’ll have to add your user to the sudoers file using /sbin/visudo 
+
+```bash
+sudo apt install git build-essential rsync cups cryptsetup efitools 
+usermod -a -G lpadmin $USER
+```
+
+Add user to sudoers file and add export `PATH=$PATH:/sbin/` to your `.bashrc`
+
+```bash
+reboot
+```
+Now create the Secure boot keys:
+
+```bash
+su -
+mkdir /etc/efi-keys
+cd /etc/efi-keys
+wget https://rodsbooks.com/efi-bootloaders/mkkeys.sh
+chmod +x mkkeys.sh
+./mkkeys.sh
+```
+For the common name, I usually enter "VotingWorks"
+
+If you're in a VM, you can skip most of the secure boot steps (the following steps). Reboot the machine and enter into the BIOS setup. Find the Secure Boot screen and select an option like "delete all keys". This should put you into "Setup Mode". Now exit out of the firmware and boot the OS. From here, persist the keys you just generated to the firmware:
+```bash
+su -
+cd /etc/efi-keys
+efi-updatevar -f DB.auth DB
+efi-updatevar -f KEK.auth KEK
+efi-updatevar -f PK.auth PK
+```
+**IMPORTANT**: the order here matters. PK must be persisted _last_, otherwise it won't work. This should work well on Lenovo devices, but if it doesn't here is an alternate approach:
+
+Copy the keys to the EFI system partition:
+```bash
+cp *.auth /boot/efi
+```
+Now reboot the machine and enter the firmware interface. On the Secure Boot screen, there should be an option to "Enroll Keys". It should show you your ESP on selection, at which point you can navigate to each of the three keys and enroll them in the firmware. As above, the order matters, so do DB, KEK then PK in that order.
+
+At this point there should be keys enrolled in firmware and the machine should be in Secure Boot mode. You will probably have to turn it off for now so that we can still boot Debian, since we haven't signed anything yet. 
+
+**IMPORTANT**: We are not using disk encryption, meaning that keys which are left on `/etc/efi-keys` or `/boot/efi` are **exposed**. The keys **must** be wiped from the disk before putting the machines in the field, preferably before cloning. 
+
```

Comment:
should we cut this out of here into a separate doc for key generation since we're now locking down based on a USB stick?

Reply:
yes, yes we should.